test/alternator/conftest.py

```diff
@@ -310,3 +310,39 @@ def optional_rest_api(dynamodb):
     except:
         return None
     return url
+
+# Fixture to check once whether newly created Alternator tables use the
+# tablet feature. It is used by the xfail_tablets and skip_tablets fixtures
+# below to xfail or skip a test which is known to be failing with tablets.
+# This is a temporary measure - eventually everything in Scylla should work
+# correctly with tablets, and these fixtures can be removed.
+@pytest.fixture(scope="session")
+def has_tablets(dynamodb, test_table):
+    # We rely on some knowledge of Alternator internals:
+    # 1. For table with name X, Scylla creates a keyspace called alternator_X
+    # 2. We can read a CQL system table using the ".scylla.alternator." prefix.
+    info = dynamodb.Table('.scylla.alternator.system_schema.scylla_keyspaces')
+    try:
+        response = info.query(
+            KeyConditions={'keyspace_name': {
+                        'AttributeValueList': ['alternator_'+test_table.name],
+                        'ComparisonOperator':  'EQ'}})
```

Comment:
I do not fully understand this expression, but note that `scylla_keyspaces`, can have an entry for a given keyspace, for other reasons too, not just when it has tablets enabled. In particular, using non-local storage (S3) will also add an entry in this table. In the future we will possibly have more such features.

Reply:
This is old-style DynamoDB key conditions (I don't know why I used the old style here, but it works).
It basically means look for a partition with `keyspace_name` equal to this expected keyspace name.

I knew that this table may contain other things too. This is why the code below also did:
```
    if 'initial_tablets' in response['Items'][0]:
        return True
    return False
```
I.e., I only return true if there's an `initial_tablets` in the response.

But now that I think about it again, maybe if the row exists, it will automatically have an `initial_tablets` but it will be a null string. Maybe I need to check not just that `initial_tablets` is in the response but also that it's a non-empty string. I now remember you did something like this in your code. 
I'll change it.

Reply:
Done.